src/developer/web-api/analytics.md

```diff
@@ -2816,6 +2816,310 @@ Table: Query parameters
 | svg | Yes | The SVG content |
 | filename | No | The file name for the returned attachment without file extension |
 
+## Analytics outlier detection { #webapi_analytics_outlier_detection } 
+
+The analytics API provides endpoints for investigation of the data quality based on Z Score and Modified Z Score. Both scores are statistical measures that help analyze and interpret data in the context of deviations from the middle value. They are particularly useful in identifying outliers or extreme values in a dataset. The API is implemented as a single analytics endpoint:
+
+- /api/analytics/outlierDetection
+
+### Request  { #webapi_analytics_outlier_detection_request } 
+
+**Query parameters**
+
+| Query parameter | Description | Required | Options (default first)
+|---|---|---|---|
+| ds | Data set | Yes | Data set identifier |
+| startDate | Start date for interval tocheck for outliers | No (relative date period is mandatory in this case) | Date (yyyy-MM-dd) |
+| endDate |End date for interval to check for outliers | No (relative date period is mandatory in this case) | Date (yyyy-MM-dd) |
+| pe | ISO periods and relative periods | No (start and end date is mandatory in this case) | see "date and period format"  |
+| relativePeriodDate | Date used as basis for relative periods. | No| Date (yyyy-MM-dd) |
+| ou | Organisation unit, organiosation unit level or groups (can be combined) | Yes |Organisation unit (level, group) identifier |
+| headers | The name of the headers to be returned as part of the response. One or more headers name separated by comma |  No | (NULL), dx, dxname, pename, pe ... |
+| orderBy | Sort the records on the value column | No | absdev, zscore, modifiedzscore, median, mean, stddev, medianabsdeviation, lowerbound, upperbound |
+| sortOrder | Sort the records on the value column in ascending or descending order | No |ASC, DESC |
+| algorithm |Algorithm to use for outlier detection | No | Z_SCORE, MODIFIED_Z_SCORE |
+| threshold | Threshold for outlier values Z_SCORE or MODIFIED_Z_SCORE| No |Numeric, greater than zero. Default: 3.0|
+| inputIdScheme | Identifier scheme to use for metadata items in the query request, can be an identifier, code or attributes. | No | UID, ID, CODE, NAME |
+| maxResults | Maximum rows (responses) | No | 500 |
+
```

Comment:
We need to include a section to describe the calculated fields, so users/clients know how the values are calculated/measured: `value`, `mean`, `medianabsdeviation`, `stddev`, `absdev`, `zscore`, `modifiedzscore`, `upperbound`, `lowerbound`.

Reply:
done